src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/MemoryLayout.java

```diff
@@ -329,9 +329,9 @@ default long bitOffset(PathElement... elements) {
      *
      * <p>The final offset returned by the method handle is computed as follows:
      *
-     * <blockquote><pre>{@code
-    offset = c_1 + c_2 + ... + c_m + (x_1 * s_1) + (x_2 * s_2) + ... + (x_n * s_n)
-     * }</pre></blockquote>
+     * {@snippet :
+     * offset = c_1 + c_2 + ... + c_m + (x_1 * s_1) + (x_2 * s_2) + ... + (x_n * s_n)
```

Comment:
not 100% sold on this being a snippet. Looking at it with javadoc, we have the "copy" button - but this is not a piece of code in any way (at the very least the language should not be java)

Reply:
Right, I reverted the change for any non Java snippet.